src/main/java/xyz/nucleoid/plasmid/game/stats/GameStatisticBundle.java

```diff
@@ -0,0 +1,57 @@
+package xyz.nucleoid.plasmid.game.stats;
+
+import com.google.gson.JsonObject;
+import it.unimi.dsi.fastutil.objects.Object2ObjectMap;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import net.minecraft.server.network.ServerPlayerEntity;
+import xyz.nucleoid.plasmid.util.PlayerRef;
+
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Wrapper containing a map players' {@link StatisticBundle}s
+ */
+public class GameStatisticBundle {
+    private final Object2ObjectMap<UUID, StatisticBundle> players = new Object2ObjectOpenHashMap<>();
+    private final StatisticBundle global = new StatisticBundle();
```

Comment:
`GameStatisticsBundle` vs `StatisticBundle` is a bit of a confusing distinction- what about renaming `StatisticBundle` -> `StatisticMap` or similar?